include/bluetooth/uuid.h

```diff
@@ -79,6 +79,58 @@ struct bt_uuid_128 {
 #define BT_UUID_32(__u) CONTAINER_OF(__u, struct bt_uuid_32, uuid)
 #define BT_UUID_128(__u) CONTAINER_OF(__u, struct bt_uuid_128, uuid)
 
+
+/**
+ * @brief Encode 128 bit UUID into an array values
+ *
+ * UUID in 128 bit form requires an array creation.
+ * To complicate the whole thing - it requires the array to start from LSB,
+ * so using the readable form, we have to write it down backwards.
+ *
+ * `6E400001-B5A3-F393-E0A9-E50E24DCCA9E` have to written like below in
+ * the code:
+ *
+ * @code
+ * 0x9E, 0xCA, 0xDC, 0x24, 0x0E, 0xE5, 0xA9, 0xE0, \
+ * 0x93, 0xF3, 0xA3, 0xB5, 0x01, 0x00, 0x40, 0x6E
+ * @endcode
+ *
+ * This macro simplifies it by the usage of more user-friendly form:
+ *
+ * @code
+ * BT_UUID_128_ENC(0x6E400001, 0xB5A3, 0xF393, 0xE0A9, 0xE50E24DCCA9E)
```

Comment:
If you keep this part you need to update to ENCODE.

Reply:
Right - done. I suppose the comment would be updated anyway, but lets keep in technically correct.